### PR TITLE
:sparkles: Add 15,30 and 60 minutes slots for onDateLongPress callback.

### PR DESCRIPTION
**Pull request by PRBaraiya**

### CODE REVIEW

File: lib/src/components/_internal_components.dart [hunk 1]
```diff
@@ -254,42 +255,57 @@ class PressDetector extends StatelessWidget {
...
+  final double heightPerMinute;
 
   /// Defines date for which events will be displayed in given display area.
   final DateTime date;
 
   /// Called when user long press on calendar.
   final DatePressCallback? onDateLongPress;
 
+  /// Defines size of the slots that provides long press callback on area
+  /// where events are not there.
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/components/_internal_components.dart:267]: typo where events are not there. > where events are not available.

File: lib/src/components/_internal_components.dart [hunk 2]
```diff
@@ -254,42 +255,57 @@ class PressDetector extends StatelessWidget {
...
+  final MinuteSlotSize slotSize;
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/components/_internal_components.dart:268]: can we name this minuteSlotSize

File: lib/src/day_view/day_view.dart [hunk 3]
```diff
@@ -129,6 +130,10 @@ class DayView<T extends Object?> extends StatefulWidget {
   /// This method will be called when user long press on calendar.
   final DatePressCallback? onDateLongPress;
 
+  /// Defines size of the slots that provides long press callback on area
+  /// where events are not there.
+  final MinuteSlotSize slotSize;
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/day_view/day_view.dart:135]: same

File: lib/src/day_view/day_view.dart [hunk 4]
```diff
@@ -156,6 +161,7 @@ class DayView<T extends Object?> extends StatefulWidget {
     this.scrollOffset = 0.0,
     this.onEventTap,
     this.onDateLongPress,
+    this.slotSize = MinuteSlotSize.minutes_60,
```
Comment (vatsaltanna-simformsolutions) [inline lib/src/day_view/day_view.dart:164]: any reason for using underscore in minutes_60?

File: lib/src/day_view/_internal_day_view_page.dart [hunk 5]
```diff
@@ -68,6 +69,10 @@ class InternalDayViewPage<T extends Object?> extends StatelessWidget {
   /// Called when user long press on calendar.
   final DatePressCallback? onDateLongPress;
 
+  /// Defines size of the slots that provides long press callback on area
+  /// where events are not there.
+  final MinuteSlotSize slotSize;
```
Comment (vatsaltanna) [inline lib/src/day_view/_internal_day_view_page.dart:74]: you missed this one

**Review by vatsaltanna-simformsolutions — APPROVED** [review]

**Pull request merged**